wxWidgets/wxWidgets
# Implementing sorted combo box on Cocoa

The initial commit to implement the sorted combo box on Cocoa.
In order to test it:

1. Compile widgets sample and execute it
2. Switch to combobox page
3. Click "Append a few strings"
4. Click "Sort items"
5. Make sure that the strings in the combo box are sorted.

Build against minimum 10.9 OSX on the 10.13 with Xcode 9.

Please review and apply.

If this is OK, I will try to fix/test Append/Insert functionality on wxComboBox with the wxCB_SORTED.

TIA!
